.github/workflows/ci.yaml

```diff
@@ -0,0 +1,52 @@
+# Defrag - centralized API for the openSUSE Infrastructure
```

Comment:
We should delete this. In general, I think a rebase on dev would be a good idea, since there are a lot of old files in here.